app/controllers/editions_controller.rb

```diff
@@ -577,7 +590,7 @@ def create_tagging_update_form_values(tagging_update_params)
   end
 
   def tagging_update_params
-    params.require(:tagging_tagging_update_form).permit(
+    tagging_update_params = params.require(:tagging_tagging_update_form).permit(
```

Comment:
Might cause confusion to have a variable with the same name as the method.
```suggestion
    update_params = params.require(:tagging_tagging_update_form).permit(
```